bc/subscription/tasks.py

```diff
@@ -38,6 +41,19 @@ def process_filing_webhook_event(fwe_pk) -> FilingWebhookEvent:
     if DO_NOT_POST.search(filing_webhook_event.description):
         return filing_webhook_event
 
+    cl_document = lookup_document_by_doc_id(filing_webhook_event.cl_doc_id)
+    document_url = (
+        f"https://storage.courtlistener.com/{cl_document['filepath_local']}"
+        if cl_document["filepath_local"]
+        else None
+    )
+
+    document = None
+    if document_url:
+        document_request = requests.get(document_url, timeout=60)
```

Comment:
You're fetching this from S3. A timeout of maybe 3 seconds would certainly be fine.

Reply:
got it!